### ISSUE TITLE
Code coverage

### ISSUE THREAD

**Issue by mjt320** [OSIPI/DCE-DSC-MRI_CodeCollection#51]
Our codecov reports look a little strange to me: only 94 source code lines tracked.
Ideally, I think we would like to measure % of contributed code tested, specifically *.py files within /src folder(?)
Grateful if anyone can help with this... Thanks.

**Comment by michaelberks** [comment]
Happy to look. Might not be before ISMRM deadline though. The code cov report I just saw when I merged my code said ~250 / 450 of lines tested over the whole repo? Though this will only be for modules imported into the test suite. I was able to get code cov on QbiPy above 90% but that meant writing a lot of unit tests to cover all the margin cases and various options. The problem is, the more complete contributed code is (in terms of offering a rich set of options), the more potential execution paths there are to cover!
